api/setup.py

```diff
@@ -91,5 +94,12 @@ def recur_expand(target_root, dir):
         print("requests >=0.12.1 is not installed", file=sys.stderr)
         sys.exit(1)
 
+    package_list = ['zulip', 'bots_api', 'bots']
+    bots_dirs = os.listdir('bots')
+    for bot in bots_dirs:
+        if os.path.isdir(os.path.join('bots', bot)):
+            package_list.append('bots.' + bot)
+    package_info['packages'] = package_list
+
 
```

Comment:
I have tested this piece of code, but not in the non-setuptools environment. I had commented the `package['packages'] = find_packages()` line in try block and had placed this code there and tested it.